spec/redux/reducer-test.js

```diff
@@ -0,0 +1,36 @@
+import loadApi from '../../src/redux/reducer';
+import * as actType from '../../src/redux/action-types';
+
+describe("The reducer", () => {
+
+  it("returns a state object", () => {
+    expect(loadApi({}, "default")).toEqual(jasmine.any(Object));
+  });
+
+  describe("returns a state change", () => {
+
+    var state = {
```

Comment:
This should go inside a beforeEach. And you can initialize it using a call to loadAPI.